pkg/services/mysql/all_in_one_provision.go

```diff
@@ -11,20 +11,19 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
-const (
-	enabled  = "enabled"
-	disabled = "disabled"
-)
-
 func (a *allInOneManager) ValidateProvisioningParameters(
+	plan service.Plan,
 	provisioningParameters service.ProvisioningParameters,
 	_ service.SecureProvisioningParameters,
 ) error {
 	pp := allInOneProvisioningParameters{}
-	if err := service.GetStructFromMap(provisioningParameters, &pp); err != nil {
+	if err := service.GetStructFromMap(
+		provisioningParameters,
+		&pp,
+	); err != nil {
```

Comment:
I'm probably going to undo this. It doesn't bother me, but since this is a big PR, I want to try and limit the changes to only those that are meaningful-- just to keep the PR as a whole more comprehensible.